src/components/TextareaField/TextareaField.stories.tsx

```diff
@@ -92,14 +91,3 @@ export const WithAMaxLength: StoryObj<Args> = {
   },
   render: (args) => <TextareaField {...args} />,
 };
-
-export const AfterDelete: StoryObj<Args> = {
```

Comment:
Main goal of this PR is to trim that one console message from appearing in the test run, and do a better and more deliberate job of testing the event handler on `TextareaField`.